.maintain/chaostest/src/hypervisor/modules/k8s.js

```diff
@@ -0,0 +1,113 @@
+const k8s = require('@kubernetes/client-node')
+const { isFunction } = require('../../utils')
+const logger = require('../../utils/logger')
+
+// load k8s
+const kc = new k8s.KubeConfig()
+kc.loadFromDefault()
+
+// load k8s Apis
+const k8sAppApi = kc.makeApiClient(k8s.AppsV1Api)
+const k8sCoreApi = kc.makeApiClient(k8s.CoreV1Api)
+
+const createNameSpace = async namespace => {
+  const namespaceJson = {
+    apiVersion: 'v1',
+    kind: 'Namespace',
+    metadata: {
+      name: namespace
+    }
+  }
+  return await k8sCoreApi.createNamespace(namespaceJson)
+}
+
+const readNameSpace = async namespace => {
+  return await k8sCoreApi.readNamespace(namespace)
+}
+
+const createPod = async (nodeSpec, namespace) => {
+  const { label, nodeId, image, args, port } = nodeSpec
+  const spec = {
+    metadata: {
+      labels: {
+        app: label
+      },
+      name: nodeId
+    },
+    spec: {
+      containers: [
+        {
+          image: image,
+          imagePullPolicy: 'Always',
+          name: nodeId,
+          ports: [{ containerPort: port }],
+          args: args
+        }
+      ]
+    }
+  }
+  return await k8sCoreApi.createNamespacedPod(namespace, spec)
+}
+
+const getDeploymentStatus = async (deploymentName, namespace) => {
+  const response = await k8sAppApi.readNamespacedDeploymentStatus(deploymentName, namespace)
+  const status = response.response.body.status
+  function getAvailability (item) {
+    return item.type === 'Available'
+  }
+  if (status && status.conditions) {
+    return status.conditions.find(getAvailability)
+  }
+  return undefined
+}
+
+const deleteNameSpace = async (namespace) => {
+  logger.debug(`Taking down NameSpace ${namespace}...`)
+  if (process.env.KEEP_NAMESPACE && process.env.KEEP_NAMESPACE === 1) {
+    return
+  }
+  return k8sCoreApi.deleteNamespace(namespace)
+}
+
+const getNameSpacedPods = async (namespace) => {
+  const response = await k8sCoreApi.listNamespacedPod(namespace)
+  return response.body.items
+}
+
+const getPod = async (nodeId, namespace) => {
```

Comment:
```suggestion
const getPod = async (podName, namespace) => {
```

To separate concerns the `k8s.js` file is only aware of Kubernetes specific terms. Thus `nodeId` is a bit misleading as it does not target a Kubernetes node (e.g. VM or bare metal machine) but a Substrate running binary. I would suggest renaming this to `podId` or `podName`.